packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -131,7 +131,7 @@ const Breakouts = WebexPlugin.extend({
           body: {rosters},
         } = result;
 
-        rosters.forEach(({locus}) => {
+        rosters?.forEach(({locus}) => {
```

Comment:
why has this changed? are rosters potentially undefined?

Reply:
Yes, It's potentially undefined in our test, just add protection to avoid JS error